cfme/utils/template/rhevm.py

```diff
@@ -78,9 +78,10 @@ def add_disk_to_vm(self):
             return
         rhevm_specs = cfme_data.template_upload.template_upload_rhevm
         disk_kwargs = dict(storage_domain=self.provider_data.template_upload.storage_domain,
-                           size=rhevm_specs.disk_size,
-                           interface=rhevm_specs.disk_interface,
-                           format=rhevm_specs.disk_format)
+                           size=rhevm_specs.get('disk_size', 5000000000),
```

Comment:
I see we are defaulting to this value, do you think it would benefit us in any way to have that as a parameter on `add_disk_to_vm`?

Reply:
Not in this case, since this is just a template upload method.